### ISSUE TITLE
Joystick Gremlin only recognises one Joystick and not my Throttle

### ISSUE THREAD

**Issue by wave004** [WhiteMagic/JoystickGremlin#496]
![grafik](https://github.com/WhiteMagic/JoystickGremlin/assets/94694696/3d6c97a1-0e2f-43b8-b7b3-7551a5cc52ad)
I have two devices plugged in - a VKB Gladiator NXT Joystick and a Thrustmaster TWCS Throttle. Both are shown to be fully functional in Windows settings, but only the Joystick shows up on JG.

I've attatched the debug.txt I've been given by the program below. I have omitted most of it as I assume the most important bits are listed here:

[2023-06-22 13:39:21.817] [debug] [info] Initializing DILL v1.3
[2023-06-22 13:39:22.224] [debug] [info] {20BCF530-B406-11EB-8001-444553540000}: Processing device:  VKB-Sim Gladiator NXT R  
[2023-06-22 13:39:22.225] [debug] [info] Skipping device {20BCF530-B406-11EB-8001-444553540000}, not yet fully initialized
[2023-06-22 13:39:22.230] [debug] [info] Device summary:  VKB-Sim Gladiator NXT R   {20BCF530-B406-11EB-8001-444553540000}
[2023-06-22 13:39:22.230] [debug] [info] Axis=8 Buttons=128 Hats=1
[2023-06-22 13:39:22.230] [debug] [info] Axis map
[2023-06-22 13:39:22.230] [debug] [info]   linear=1 id=1
[2023-06-22 13:39:22.230] [debug] [info]   linear=2 id=2
[2023-06-22 13:39:22.230] [debug] [info]   linear=3 id=3
[2023-06-22 13:39:22.230] [debug] [info]   linear=4 id=4
[2023-06-22 13:39:22.230] [debug] [info]   linear=5 id=5
[2023-06-22 13:39:22.230] [debug] [info]   linear=6 id=6
[2023-06-22 13:39:22.230] [debug] [info]   linear=7 id=7
[2023-06-22 13:39:22.230] [debug] [info]   linear=8 id=8
[2023-06-22 13:39:22.241] [debug] [error] {20BCF530-B406-11EB-8001-444553540000}: Attempted to read buffered device data from a device that is not buffered. (DIERR_NOTBUFFERED)
[2023-06-22 13:39:22.241] [debug] [error] {20BCF530-B406-11EB-8001-444553540000} Failed reading device in buffered mode, falling back to polling, Attempted to read buffered device data from a device that is not buffered. (DIERR_NOTBUFFERED)

I don't know how to "fully initialize" the TWCS.

**Comment by WhiteMagic** [comment]
You should provide the full log, as the above actually is missing crucial information. For some reason the TWCS isn't complying with buffered reading mode and DILL has to fallback to a cruder way of reading information. However, the parts that would follow in the log are missing and would be crucial to see if that fallback mode actually works. By the looks of it it doesn't but I can't tell why.

**Comment by wave004** [comment]
Right, sorry. Full log is too long for me to post on the site in text form, but I can give you the .txt file:

[debug.txt](https://github.com/WhiteMagic/JoystickGremlin/files/11834096/debug.txt)

**Comment by WhiteMagic** [comment]
Looking at the log there seem to be quite a few things off but nothing in there suggests that the TWCS is completely dropped. If you only connect the TWCS does it show up?

**Comment by wave004** [comment]
Nope! with just the TWCS connected it only displays this:
![grafik](https://github.com/WhiteMagic/JoystickGremlin/assets/94694696/12375ea4-1e12-4437-9b8e-593ccaefbe42)
debug.txt:
[debug.txt](https://github.com/WhiteMagic/JoystickGremlin/files/11837748/debug.txt)

with both connected and vjoy + joystick gremlin reinstalled this is what shows up:
[debug.txt](https://github.com/WhiteMagic/JoystickGremlin/files/11837765/debug.txt)

**Comment by upintilldawn** [comment]
i have the same problem with virpil panel #2. in windows but this program does not see it. i also have a problem when i change vjoy.. give a strange erro.. not sure where error logs are keeped.
